backend/src/model/Review.ts

```diff
@@ -0,0 +1,29 @@
+import mongoose, { Schema, Types } from "mongoose";
+
+export interface IReview {
+  title: string;
+  description: string;
+  sentiment: number;
```

Comment:
Since sentiment is not a required value, it should be optional in the interface or else typescript will complain when its undefined
```suggestion
  sentiment?: number;
```